js/textfieldCounter.js

```diff
@@ -1,14 +1,20 @@
 //Limit textfield for create question to 200
 
 export function counterTextField() {
-  const textAreaCounter = document.getElementById('yolo');
-
-  const textFieldCreateElement = document.querySelector(
-    '[data-js="add-question"]'
+  const questionInputElement = document.querySelector(
+    '[data-js="add-question-input"]'
   );
+  const outputElement = document.querySelector('[data-js="outputCounter"]');
 
-  textFieldCreateElement?.addEventListener('input', event => {
-    const result = 200 - textFieldCreateElement.value.length;
-    textAreaCounter.textContent = `Please enter your question:   -- Characters left: ${result}`;
+  questionInputElement?.addEventListener('input', event => {
```

Comment:
i think, you dont need the argument `event`here as you dont use it in the function itself, you can make the call anonymous with ()